motioneye/static/js/main.js

```diff
@@ -528,10 +528,25 @@ function showErrorMessage(message) {
 }
 
 function doLogout() {
-    setCookie(USERNAME_COOKIE, '_');
+    setCookie(USERNAME_COOKIE, '');
+    setCookie(PASSWORD_COOKIE, '');
     window.location.reload(true);
 }
 
+function isAuthCookiesSet() {
```

Comment:
Since it's two cookies, plural 😛 (while I know that `is` is a pretty nice prefix for functions returning boolean values, so feel free to ignore my grammatical comment 😅):
```suggestion
function AreAuthCookiesSet() {
```